chaoscenter/web/src/components/ExperimentActionButtons/ExperimentActionButtons.tsx

```diff
@@ -394,11 +396,13 @@ export const EnableDisableCronButton = ({
     intent: Intent.WARNING,
     onClose: (isConfirmed: boolean) => {
       if (isConfirmed) {
+        const diable = isCronEnabled ? true : false;
```

Comment:
typo in word disable, also value of disable will always be same as isCronEnabled, so code can be modified to just add const disable = isCronEnabled